packages/shared/src/components/Feed.tsx

```diff
@@ -202,27 +180,11 @@ export default function Feed<T>({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [emptyFeed]);
 
-  const showScrollOnboardingVersion =
-    sidebarRendered &&
-    feedName === MainFeedPage.Popular &&
-    !isLoading &&
-    alerts?.filter &&
-    !user?.id &&
-    onboardingV2 === OnboardingV2.Control;
-
   const infiniteScrollRef = useFeedInfiniteScroll({
     fetchPage,
-    canFetchMore: canFetchMore && !showScrollOnboardingVersion,
```

Comment:
This is still ongoing, right? Along with the post modal filter experiment, they are considered the `control` value of the `onboarding_v2`. Or I am probably wrong.

Reply:
Wait! It should be removed only if the onboarding wall succeeds. Not yet

Reply:
According to Sab it interferes with the experiment and should have been removed 😅

Reply:
We will keep it, so i will revert those changes 🙏